src/pages/Jongmin/Component/CoffeeCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./CoffeeCard.scss";
+
+function CoffeeCard(props) {
+  // useNavigate를 이용한 방법
+  //   const navigate = useNavigate();
+  //   const goToList = () => {
+  //     navigate("/detail-jongmin");
+  //   };
+  //   return (
+  //     <div className="CoffeeCard">
+  //       <div className="imgHover">
+  //         <img alt="{props.name}" src={props.image} onClick={goToList} />
+  //       </div>
+  //       <p className="coffeeTitle">{props.name}</p>
+  //     </div>
+  //   );
+
+  // 좋아요(하트) 클릭 기능 구현
+  const [onHeart, setOnHeart] = useState("🤍");
+  const heartSelection = () => {
+    if (onHeart === "❤️") {
+      return "🤍";
+    } else if (onHeart === "🤍") {
+      return "❤️";
+    }
+  };
```

Comment:
else if 대신 else로 사용하시면 훨씬 간단해질 거예요.
```suggestion
  const heartSelection = () => {
    if (onHeart === "❤️") {
      return "🤍";
    } 
    else {
      return "❤️";
    }
  };
```